events/code_review_events/bugbug_utils.py

```diff
@@ -132,15 +130,14 @@ def should_run_risk_analysis(self, build):
         if self.community_tc is None:
             return False
 
-        reviewerPHIDs = [
+        userPHIDs = [
             reviewer["reviewerPHID"]
             for reviewer in build.revision["attachments"]["reviewers"]["reviewers"]
         ]
 
-        return any(
-            reviewerPHID in self.risk_analysis_reviewers
-            for reviewerPHID in reviewerPHIDs
-        )
+        userPHIDs.append(build.revision["fields"]["authorPHID"])
```

Comment:
Nit: i would simply add a comment around here saying that we run BugBug for both reviewers & authors

Reply:
I've updated the docstring of the function